### ISSUE TITLE
SyntaxError: Unexpected token %%wrapName%%

### ISSUE THREAD

**Issue by PierreGUI** [callstack/linaria#675]
## Environment
- Linaria version: 1.4.0-rc.1
- Bundler (+ version): webpack 4.43
- Node.js version: 12.18
- OS: Mac OS X

## Description
**1.4.0-rc.0** works fine!  
Just updated to **1.4.0-rc.1** (tried 3, 2, 1) and getting this error in any file with a function interpolation:
```
ERROR in ./src/components/clients/pages/layout/Pricing.tsx
Module build failed (from ./node_modules/linaria/loader.js):
SyntaxError: Unexpected token (3:8)
  1 | /* @babel/template */;
  2 |
> 3 |   const %%wrapName%% = (fn) => {
    |        ^
  4 |     try {
  5 |       return fn();
  6 |     } catch (e) {
    at Parser.raise (.../client/node_modules/@babel/parser/lib/index.js:4021:15)
    at Parser.unexpected (.../client/node_modules/@babel/parser/lib/index.js:5330:16)
    at Parser.parseBindingAtom......
```

## Reproducible Demo
```js
import { styled as linaria } from 'linaria/react'

export function zi (index) { return ZIndex[index] }

const ToastsWrapper = linaria.div`
	left: 0;
	position: fixed;
	right: 0;
	top: 0;
	z-index: ${zi(2)};
`
```

**Comment by Anber** [comment]
Hi @PierreGUI 
Your reproducible demo seems correct and works well in the test environment. Could you please provide a sample repo with this error?

**Comment by PierreGUI** [comment]
Hey @Anber thanks for your answer. I've spent some time trying to make a sample repo (forking my own current repo) but there the bug was not showing up. I came to the conclusion the issue has something to do with my NPM packages. I can't seem to identify which one exactly, it works after I removed my `package-lock` file and ran `npm install` again...  Not quite the solution I was hoping for 👍   
I'll keep you posted, but I think we can close in the meantime.
